### PR TITLE
from Python2 to Python3

### PR DESCRIPTION
**Pull request by chengshifan**

### CODE REVIEW

File: ib.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def execute(command, cwd=None, env=None, ignore_error=False, ignore_output=False
         if ignore_output:
             continue
 
-        print(line.rstrip())
+        print((line.rstrip()))
```
Comment (iberryful) [inline ib.py:68]: useless parenthesis

File: infrabox/test/api/job_api_test.py [hunk 2]
```diff
@@ -237,6 +237,6 @@ def test_testresult(self):
         keys = ['status', 'duration', 'message', 'stack']
         for i, received_row in enumerate(r):
             # create dictionary from the list to compare it easier
-            row_dictionary = dict(zip(keys, received_row))
-            self.assertTrue(all(item in testresult_data["tests"][i].items()
-                                for item in row_dictionary.items()))
+            row_dictionary = dict(list(zip(keys, received_row)))
+            self.assertTrue(all(item in list(testresult_data["tests"][i].items())
```
Comment (iberryful) [inline infrabox/test/api/job_api_test.py:241]: useless wrapper

File: infrabox/test/e2e/test.py [hunk 3]
```diff
@@ -73,7 +73,7 @@ def _get_build(self):
             return result.json()[0]
         except:
             print("Get build failed: ")
-            print(result.text)
+            print((result.text))
```
Comment (iberryful) [inline infrabox/test/e2e/test.py:76]: useless parenthesis

File: infrabox/test/e2e/test.py [hunk 4]
```diff
@@ -322,7 +322,7 @@ def main():
     print("Connecting to DB")
     connect_db() # Wait for DB
 
-    print("ROOT_URL: %s" % root_url)
+    print(("ROOT_URL: %s" % root_url))
```
Comment (iberryful) [inline infrabox/test/e2e/test.py:325]: useless parenthesis

File: infrabox/test/e2e/test.py [hunk 5]
```diff
@@ -332,7 +332,7 @@ def main():
             if r.status_code in (200, 404):
                 break
 
-            print(r.text)
+            print((r.text))
```
Comment (iberryful) [inline infrabox/test/e2e/test.py:335]: useless parenthesis

File: infrabox/test/e2e/tests/docker_compose_job/tests/tests.py [hunk 6]
```diff
@@ -3,5 +3,5 @@
 def test_get():
     print("get")
     r = requests.get("http://test-server:3000")
-    print(r.json())
+    print((r.json()))
```
Comment (iberryful) [inline infrabox/test/e2e/tests/docker_compose_job/tests/tests.py:6]: useless parenthesis

**Review by iberryful — CHANGES_REQUESTED** [review]

File: ib.py (hunk 1 above)
Comment (chengshifan) [reply ib.py]: done

**Pull request closed without merging**